apps/web/src/components/ScreeningDecisions/ScreeningDecisionDialog.tsx

```diff
@@ -0,0 +1,438 @@
+import * as React from "react";
+import { useIntl } from "react-intl";
+import { SubmitHandler } from "react-hook-form";
+
+import {
+  AssessmentStep,
+  CreateAssessmentResultInput,
+  Experience,
+  PoolCandidate,
+  Skill,
+  SkillCategory,
+  UpdateAssessmentResultInput,
+  UserSkill,
+} from "@gc-digital-talent/graphql";
+import {
+  Accordion,
+  Button,
+  Dialog,
+  HeadingLevel,
+  Well,
+  incrementHeadingRank,
+} from "@gc-digital-talent/ui";
+import { BasicForm, Submit } from "@gc-digital-talent/forms";
+import { notEmpty } from "@gc-digital-talent/helpers";
+import {
+  commonMessages,
+  getAssessmentStepType,
+  getBehaviouralSkillLevel,
+  getBehaviouralSkillLevelDefinition,
+  getLocalizedName,
+  getTechnicalSkillLevel,
+  getTechnicalSkillLevelDefinition,
+} from "@gc-digital-talent/i18n";
+import { toast } from "@gc-digital-talent/toast";
+
+import { getExperienceSkills } from "~/utils/skillUtils";
+import {
+  useCreateAssessmentResultMutation,
+  useUpdateAssessmentResultMutation,
+} from "~/api/generated";
+
+import useLabels from "./useLabels";
+import ExperienceCard from "../ExperienceCard/ExperienceCard";
+import useDialogType, { DialogType } from "./useDialogType";
+import ScreeningDecisionDialogForm, {
+  FormValues,
+} from "./ScreeningDecisionDialogForm";
+import useHeaders from "./useHeaders";
+import {
+  convertFormValuesToApiCreateInput,
+  convertFormValuesToApiUpdateInput,
+  FormValuesToApiCreateInputArgs,
+  FormValuesToApiUpdateInputArgs,
+} from "./utils";
+
+const AssessmentStepTypeSection = ({
+  type,
+  userSkill,
+  classificationGroup,
+}: {
+  type: DialogType;
+  userSkill?: UserSkill;
+  classificationGroup?: string;
+}) => {
+  const intl = useIntl();
+  switch (type) {
+    case "EDUCATION":
+      return (
+        <div>
+          <p>
+            {intl.formatMessage({
+              defaultMessage: "Selected requirement option:",
+              id: "FS4Dg5",
+              description:
+                "Header for selected requirement option in education requirement screening decision dialog.",
+            })}
+          </p>
+          <Well
+            data-h2-margin-bottom="base(x1)"
+            data-h2-text-align="base(left)"
+          >
+            {classificationGroup === "EX"
+              ? intl.formatMessage({
+                  defaultMessage:
+                    "minimum experience or education requirement (graduation with degree)",
+                  id: "Up2D3q",
+                  description:
+                    "Message on education requirements section of screening decision dialog.",
+                })
+              : intl.formatMessage({
+                  defaultMessage:
+                    "minimum experience or education requirement (2 years of post-secondary)",
+                  id: "kCbOcK",
+                  description:
+                    "Message on education requirements section of screening decision dialog.",
+                })}
+          </Well>
```

Comment:
Copy slightly mismatched? Capitalization too

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/5aa4ddcd-0f46-4a79-b412-4a1c1813744e)

Reply:
@Jerryescandon What should be displayed for the **Selected requirement option**. I thought we wanted to match the text from the Application review page:

  - Minimum experience or education requirement (graduation with degree) --> for EX classification
  - Minimum experience or education requirement (2 years of post-secondary)
  
 Is that correct?

Reply:
Okay @vd1992, after chatting with Jerbo we settled on adding the text that is selected (the radio group) by the user in the Application Education step (e.g. I meet the applied work experience option). There is a [function](https://github.com/GCTC-NTGC/gc-digital-talent/blob/8397-screening-decision-dialogs/apps/web/src/pages/Applications/ApplicationEducationPage/utils.tsx#L97) I made previously that provides the text depending on the context (e.g. classificationGroup, isIAP, etc.). Should be fixed by [9a6fa64](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8653/commits/9a6fa6440e5f9a38d08eabef9df9e641ccfccad5)